frontend/src/components/Cell.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Badge from "./Badge";
+
+export default (props: {
```

Comment:
Should define the component prop types and component name here!

Reply:
Got it, fixed it.